src/XMakeTasks/FileIO/WriteLinesToFile.cs

```diff
@@ -23,6 +17,7 @@ public class WriteLinesToFile : TaskExtension
         private ITaskItem[] _lines = null;
         private bool _overwrite = false;
         private string _encoding = null;
+        private static Encoding s_defaultEncoding = new UTF8Encoding(false, true);
```

Comment:
Recommend using named arguments here.  On a casual read no idea what `false` and `true` mean here.